Draft revision to powers & responsibilities (#230)
carpentries/trainers: powers_responsibilities.md

## Powers

### What powers belong to the Trainer community?
* Ratify documentation changes that:
     * substantially change requirements or process for maintaining Active status as a Trainer

>      * reduce or re-assign powers that belong to the Trainer community

Author: Add "are requested for ratification by any active Trainer"

Author: Also reference governance document for process

Author: remove "substantially"

>      * substantially change requirements or process for maintaining Active status as a Trainer
>      * reduce or re-assign powers that belong to the Trainer community

Author: ```suggestion
     * change requirements or process for maintaining Active status as a Trainer
     * reduce or re-assign powers that belong to the Trainer community
     * are requested for ratification by any active Trainer
     * for more on ratification, see [governance documentation](governance.md)
     
```

* Elect Trainers Leadership Committee
* Communicate with Trainers, Trainers Leadership Committee & Core Team
* Set agenda for Trainer meetings
* Volunteer or nominate Trainers for elected and appointed roles
* Propose changes to policy or documentation to be considered by Trainers Leadership

### What powers belong to the Trainers Leadership Committee?
* Interpret requirements for maintaining Active status as a Trainer, including determination of "equivalent activities" to renew via participation
* Rank Instructor Trainer applicants 
* Set schedule for Trainer meetings
* Establish Task Forces and Subcommittees within Trainers group
* Propose documentation changes for ratification by the Trainer community (see above) 
* Propose and implement documentation changes that do not require ratification by the Trainer community
* Propose changes to Core Team policies and procedures related to Instructor Training

> * Propose changes to other community practices related to Instructor Training (e.g. Maintainers)

Reviewer: Since we're talking about another community here, maybe we could be more explicit with the sort of changes we might propose for Maintainers community. Noting _who_ we would propose those changes to would also be warranted.

> * Propose changes to other community practices related to Instructor Training (e.g. Maintainers)

Author: ```suggestion
* Propose changes to policies or practices governed by other subcommunities' leadership that affect Instructor Training (e.g. Maintainers responses to checkout contributions)
```

* Oversee changes to Instructor Training curriculum
* Oversee changes to Instructor checkout requirements
* Oversee changes to criteria for ranking Open Instructor Training applicants and selection of groups for Sponsored Instructor Training

### What powers belong to the Core Team?
* Propose changes to policy or documentation to be considered by Trainers Leadership
* Accept Instructor Trainer applicants
* Accept Open Instructor Training applicants
* Award sponsored Instructor Training seats or events to groups
* Approve Instructor Training events

> * Public outreach, fundraising, and other public-facing activities to be conducted under The Carpentries brand 

Reviewer: * Conduct public outreach, fundraising, and other public-facing activities under The Carpentries brand
(revised for consistency with other point starting with verb)

> * Public outreach, fundraising, and other public-facing activities to be conducted under The Carpentries brand 

Author: ```suggestion
* Conduct public outreach, fundraising, and other public-facing activities to be conducted under The Carpentries brand 
```

> * Communications with member organisations, including negotiation of additional responsibilities for Trainers serving in the context of a Member Agreement

Reviewer: * Communicate with member organizations, including negotiations of additional responsibilities for Trainers serving in the context of a Member Agreement

> * Communications with member organisations, including negotiation of additional responsibilities for Trainers serving in the context of a Member Agreement

Author: ```suggestion
* Communicate with member organisations, including negotiation of additional responsibilities for Trainers serving in the context of a Member Agreement
```

> * Data collection, assessment, and database management pursuant to local and global privacy regulations

Reviewer: * Perform data collection, assessment, and database management pursuant to local and global privacy regulations

> * Data collection, assessment, and database management pursuant to local and global privacy regulations

Author: ```suggestion
* Perform Data collection, assessment, and database management pursuant to local and global privacy regulations
```

> * Certifying Trainers

Reviewer: Certify Trainers

> * Certifying Trainers

Author: ```suggestion
* Certify Trainers
```

> * Implementing Active status renewal and changes to or from Alumni status for Trainers

Reviewer: * Implement Active status renewal and changes to or from Alumni status for Trainers

> * Implementing Active status renewal and changes to or from Alumni status for Trainers

Author: ```suggestion
* Implement Active status renewal and changes to or from Alumni status for Trainers
```
